rabbitmq/plugins.sls

```diff
@@ -0,0 +1,31 @@
+{% from "rabbitmq/map.jinja" import rabbitmq with context %}
+
+include:
+  - .configure
+  - .install
+
+{% for plugin in rabbitmq.get('plugins', []) %}
+{% if plugin.get('package_name') %}
+download_rabbitmq_plugin_{{ plugin.name }}:
+  file.managed:
+    - name: /usr/lib/rabbitmq/lib/rabbitmq_server-{{ rabbitmq.version.split('-')[0] }}/plugins/{{ plugin.package_name }}
+    - source: https://www.rabbitmq.com/community-plugins/v3.6.x/{{ plugin.package_name }}
```

Comment:
Hmm do you think it'd be safe to pull out the `major.minor` section from `rabbitmq.version` here and plop it in this URL like `/v{{ ...cutout... }}.x/`? My thought is it would be a more explicit failure on incompatible plugins if it fails to download plugins on these steps after an upgrade.